app/views/patients/_patient_information.html.erb

```diff
@@ -40,7 +40,13 @@
           </div>
         </div>
 
-        <%= f.text_field :county, autocomplete: 'off' %>
+        <% if county_options.present? %>
+          <%= f.select :county, options_for_select(county_options(patient.county),
+                                        patient.county), autocomplete: 'off' %>
+        <% else %>
+          <%= f.text_field :county, autocomplete: 'off' %>
+        <% end %>
+
```

Comment:
--> new view code

Reply:
well handled I think! maybe a system test confirming this works, but I don't think that's completely necessary and defer to you.